ansible_3par_docker_plugin/README.md

```diff
@@ -54,6 +54,7 @@ These playbooks perform the following tasks on the Master/Worker nodes as define
       | ```hpe3par_iscsi_ips```  | No  | No default value | Comma separated iscsi IPs. If not provided, all iscsi IPs will be read from the array and populated in hpe.conf |
       | ```vlan_tag```  | No  | False | Populates the iscsi_ips which are vlan tagged, only applicable if ```hpe3par_iscsi_ips``` is not specified |
       | ```replication_device```  | No  | No default value | Replication backend properties |
+      | ```dory_installer_version```  | No  | dory_installer_v32 | Required for Openshift/Kubernetes setup.Dory installer version, supported versions are dory_installer_v31,dory_installer_v32 |
```

Comment:
@bhagyashree-sarawate Please add a space after full stop and comma. Rest looks good to me.